### ISSUE TITLE
when a table is copied from excel into word, NVDA fails to detect when you enter a table.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#5927]
This package has an excel with two sheets, and a word doc with one already pasted in.
1. Write a bunch of text into a word doc.
2. Paste one of the excel tables into a word doc.
3. do the same and paste the other table in.
4. create a new table using word.
5. fill it in with junk.
6. switch to browse mode.
7. go to the top and read.
8. Once you're in a table with data from Excel, NVDA doesn't say anything. Use table nav, it works as  expected.
9. Find the native table. Notice that all is fine.

**Comment by derekriemer** [comment]
[browse_mode_bug.zip](https://github.com/nvaccess/nvda/files/251739/browse_mode_bug.zip)

**Comment by derekriemer** [comment]
Note that both of these bugs I reported with this document were discovered with real data from a biology lab, and aren't just me trying to exercise NVDA's capabilities.

**Comment by jcsteh** [comment]
This is because it gets treated as a layout table. A layout table is currently defined as a table with no borders, and I guess this must be true for Excel tables. This was implemented to prevent spurious announcement of layout tables in Outlook emails.

Any thoughts on how we can restrict this further, @MichaelDCurran?

**Comment by michaelDCurran** [comment]
First thing that comes to mind is to restrict this specifically to Outlook browsemode. Shouldn't be too hard. Other than that we'd need to come up with a better heuristic for layout tables.
